Mathlib/Analysis/InnerProductSpace/Adjoint.lean

```diff
@@ -344,15 +365,15 @@ theorem adjoint_eq_toCLM_adjoint (A : E →ₗ[𝕜] F) :
 
 /-- The fundamental property of the adjoint. -/
 theorem adjoint_inner_left (A : E →ₗ[𝕜] F) (x : E) (y : F) : ⟪adjoint A y, x⟫ = ⟪y, A x⟫ := by
-  haveI := FiniteDimensional.complete 𝕜 E
-  haveI := FiniteDimensional.complete 𝕜 F
+  have := FiniteDimensional.complete 𝕜 E
+  have := FiniteDimensional.complete 𝕜 F
```

Comment:
This is okay as you have it (i.e., `have` is better), because it should be a Lean3-ism due to porting.